### PR TITLE
Update AGGREGATE.md

### PR DESCRIPTION
**Pull request by csharrison**
This PR updates some API shape and functionality for the aggregate proposal.

### CODE REVIEW

File: AGGREGATE.md [hunk 1]
```diff
@@ -39,72 +33,312 @@
...
+reported to the reporting origin at the path
+`.well-known/attribution-reporting/report-aggregate-attribution`.
+
+The report itself does not contain histogram contributions in the clear. Rather,
+the report embeds them in an _encrypted payload_ that can only be read by a
+trusted aggregation service known by the browser.
+
+The report will be JSON encoded with the following scheme:
+
+```json
```
Comment (johnivdel) [inline AGGREGATE.md:186]: drop

File: AGGREGATE.md [hunk 2]
```diff
@@ -39,72 +33,312 @@
...
+
+    // Purchase was for $52. The site's max value is $1024.
+    // $1 = (L1 / 2) / 1024.
+    // $52 = 52 * (L1 / 2) / 1024 = 1664 
+    "geoValue": 1664
+}
+```
+
+The scheme above will generate the following abstract histogram contributions:
+```json
```
Comment (johnivdel) [inline AGGREGATE.md:149]: drop

File: AGGREGATE.md [hunk 3]
```diff
@@ -39,72 +33,312 @@
...
+]
+```
+The other new header `Attribution-Reporting-Register-Aggregatable-Values` lists
+an amount of an abstract "value" to contribute to each key, which can be
+integers in [1, 2^16). These are attached to aggregation keys in the order they
+are generated. See the [contribution
+budgeting](#contribution-bounding-and-budgeting) section for more details on how
+to allocate these contribution values.
+
+```json
```
Comment (johnivdel) [inline AGGREGATE.md:131]: drop

**Review by johnivdel — APPROVED** [review]
LGTM % rendering comment

File: AGGREGATE.md [hunk 4]
```diff
@@ -39,72 +33,312 @@
...
+prefix. Also consider using a single parameter to specify the key vs. a
+string piece and an offset.
 
 ### Attribution trigger registration
 
-At trigger time, we have a
+Trigger registration will also include two new headers:
+`Attribution-Reporting-Register-Aggregatable-Trigger-Data` is a list of dict
+which generates aggregation keys.
+```json
```
[the remark this answers is not in the document]
Comment (csharrison) [reply AGGREGATE.md:107]: Done

File: AGGREGATE.md (hunk 1 above)
Comment (csharrison) [reply AGGREGATE.md:186]: Done

File: AGGREGATE.md (hunk 2 above)
Comment (csharrison) [reply AGGREGATE.md:149]: Done

**Pull request merged**